packages/discovery-provider/src/queries/get_followees_for_user.py

```diff
@@ -10,14 +10,16 @@
     followee_user_id
 from
     follows
-    left outer join aggregate_user on followee_user_id = user_id
+    join users on users.user_id = follows.followee_user_id
+    left outer join aggregate_user on followee_user_id = aggregate_user.user_id
```

Comment:
for future reader's sake, i don't believe this left outer is necessary. aggregate user should always exist. but it's probably fine 🤷